ospf6d/ospf6_gr_helper.c

```diff
@@ -982,6 +982,11 @@ CPP_NOTICE("Remove JSON object commands with keys starting with capital")
 						 .last_exit_reason]);
 		}
 
+		if (ospf6->ospf6_helper_cfg.active_restarter_cnt)
```

Comment:
Can't it be printed unconditionally?

Reply:
Yes , it can be. But here, it is intended to display only when there is active number of RESTARTER peers exists. do you suggest to display even there is no active RESTARTERS ?